server/src/main/scala/org/http4s/server/middleware/Logger.scala

```diff
@@ -17,11 +17,12 @@ object Logger {
   def apply[F[_]: Effect](
                            logHeaders: Boolean,
                            logBody: Boolean,
-                           redactHeadersWhen: CaseInsensitiveString => Boolean = Headers.SensitiveHeaders.contains
+                           redactHeadersWhen: CaseInsensitiveString => Boolean = Headers.SensitiveHeaders.contains,
+                           uriHeaderName: String = "uri"
                          )
                          (httpService: HttpService[F]): HttpService[F] =
-    ResponseLogger(logHeaders, logBody)(
-      RequestLogger(logHeaders, logBody)(
+    ResponseLogger(logHeaders, logBody, redactHeadersWhen, uriHeaderName)(
```

Comment:
I think not passing `redactHeadersWhen` here was a bug.

Reply:
I think you're right.